language/stdlib/modules/Diem.move

```diff
@@ -509,10 +556,24 @@ module Diem {
     public fun preburn_to<CoinType>(
         account: &signer,
         coin: Diem<CoinType>
-    ) acquires CurrencyInfo, Preburn {
+    ) acquires CurrencyInfo, Preburn, ConcurrentPreburns { // how will this work
```

Comment:
acquires 'could access in global storage' (so not an issue for old DDs without this)